apps/backend/src/app/cloudinary/cloudinary.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@nestjs/common';
+import { v2 as cloudinary } from 'cloudinary';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class CloudinaryService {
+  constructor(private readonly configService: ConfigService) {
+    cloudinary.config({
+      cloud_name: this.configService.get<string>('cloudinary.cloudName'),
+      api_key: this.configService.get<string>('cloudinary.apiKey'),
+      api_secret: this.configService.get<string>('cloudinary.apiSecret'),
+    });
+  }
+  async uploadImage(
+    image: any
+  ): Promise<{ secureUrl: string; publicId: string }> {
```

Comment:
Create new type for { secureUrl: string; publicId: string }, so you can re-use it